org.aposin.licensescout.core/src/main/resources/templates/license_report_html_default.vm

```diff
@@ -1,309 +1,309 @@
-
-<table id="license_table">
-	<thead>
-		<tr>
-			<th>Library</th>
-			<th>Version</th>
-			<th>Detection Status</th>
-			<th>Legal Status</th>
-			<th>Licenses</th>
-			#if($reportConfiguration.showDocumentationUrl)
-			<th>Documentation</th>
-			#end
-			#if($reportConfiguration.showPathColumn)
-			<th>Path</th>
-			#end
-			#if($reportConfiguration.showMessageDigestColumn)
-			<th>Message digest ($messageDigestAlgorithm)</th>
-			#end
-			#if($reportConfiguration.showLicenseCandidateFilesColumn)
-			<th>Candidate License files</th>
-			#end
-		</tr>	
-	</thead>
-<tbody>
-#foreach($archive in $archiveFiles)
-
-<tr>
-	#if (($velocityCount % 2) == 0)
-	#set ($count = 0)
-	#else
-	#set ($count = 1)
-	#end
-	
-	#set ($style = "row_" + $count)
-
-	<td class="$style"><span title="$archive.path">$archive.fileName</span></td>
-	<td class="$style"><span title="$archive.version">$archive.version</span></td>
-	<td class="$style"><span title="$archive.detectionStatus">$archive.detectionStatus</span></td>
-	<td class="$style"><span title="$archive.legalStatus">$archive.legalStatus</span></td>
-	<td class="$style" style="white-space:normal">
-		#foreach($license in $archive.licenses)
-		
-			#set ($files = "<ul>")
-			#foreach($path in $archive.getFilePaths($license))
-			#set ($files = $files + "<li>" + $path + "</li>")
-			#end
-			#set ($files = $files + "</ul>")
-		
-			#set ($td_Id = $td_Id + 1)
-			#set ($myId = "td_" + $td_Id)
-			#set ($myId_license_name = "td_" + $td_Id + "_license_name")
-		
-			<a onclick="ShowHide('$myId'); return false;" href="#"><span id="$myId_license_name" style="font-weight:normal">$license.name
-			#if ($license.versionPresent)			
-				Version $license.version			
-			#end
-			</span>
-			</a>
-			<br/>
-			<div style="display: none; padding-left: 30px;" id="$myId">
-				<p>
-					<a href="$license.urlPublic" title="$license.name" target="_blank">Public License URL</a>
-					#if($license.urlLocal)
-						&nbsp;<a href="./lizenzen/$license.urlLocal" title="$license.name" target="_blank">Local License URL</a>
-					#end	
-				</p>
-				<p><b>Referencing Files</b></p>
-				$files
-			</div>
-			<br/>	
-		#end
-	</td>
-	#if($reportConfiguration.showDocumentationUrl)
-	<td class="$style">
-	#if($archive.documentationUrl &&  !$archive.documentationUrl.empty)
-	<a href="$archive.documentationUrl" title="$archive.documentationUrl" target="_blank">Documentation</a>
-	#end
-	</td>
-	#end
-	#if($reportConfiguration.showPathColumn)
-	<td class="$style"><span title="$archive.path">$archive.path</span></td>
-	#end
-	#if($reportConfiguration.showMessageDigestColumn)
-	<td class="$style"><span title="$archive.messageDigestString">$archive.messageDigestString</span></td>
-	#end
-	#if($reportConfiguration.showLicenseCandidateFilesColumn)
-	<td class="$style">
-		#foreach($candidateFilePath in $archive.licenseCandidateFiles)
-		$candidateFilePath <br/>
-		#end
-	</td>
-	#end
-</tr>
-#end
-</tbody>
-
-</table>
-
-
-</body>
-</html>
+#*

```

Comment:
Please double check this file. It looks there is a unwanted line ends change that makes the whole file marked as changed.

Reply:
I fixed the line endings.